soh/src/overlays/actors/ovl_En_Mk/z_en_mk.h

```diff
@@ -21,4 +21,6 @@ typedef struct EnMk {
     /* 0x0284 */ EnMkActionFunc actionFunc;
 } EnMk; // size = 0x0288
 
+void EnMk_Wait(EnMk* enMk, PlayState* play);
```

Comment:
Please defer to forward declaring this function in the .cpp extern block so that we don't have to modify the src actor header.

Reply:
Was just copying an existing hook, done.